_posts/2023-04-10-opensource-perf.markdown

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title:  "Collaborating to Improve Open Source Performance: A Case Study"
```

Comment:
I appreciate your calling out our collaboration, but you did all the work! I would just call it "Improving JSON parsing performance in opensearch-java" and removing this "dblock" person from the authors!